### PR TITLE
UCT/RDMACM: fix error flow in iface init.

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

**Comment by alinask** [comment]
might be related to https://github.com/openucx/ucx/issues/4354

File: src/uct/ib/rdmacm/rdmacm_iface.c [hunk 1]
```diff
@@ -580,7 +581,9 @@ static UCS_CLASS_INIT_FUNC(uct_rdmacm_iface_t, uct_md_h md, uct_worker_h worker,
     return UCS_OK;
 
 err_destroy_id:
+if (self->is_server) {
```
Comment (yosefe) [inline src/uct/ib/rdmacm/rdmacm_iface.c]: indentation

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>UNKNOWN</b> on 24 workers (click for details)</summary>

**Note:** the logs will be deleted after 14-Nov-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/ff6f33396b109f5e2ac4a037aebe5a276aab4b76/_main) | :question: ABORTED
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/738eb4e143a733a3c353ae830721bf6ec246c955/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/5c5ef07c479f4799617a45d0f84aeefc34a07339/hpc-arm-cavium-jenkins_W0) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/234639cac385bf6a60808f977855f5da07fa9db3/hpc-arm-cavium-jenkins_W1) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/39d186a1225df12e7c702dcf103ae49f5be75026/hpc-arm-cavium-jenkins_W2) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/9c5bb047131d9c59d266e960ecf53847b2aa8d81/hpc-arm-cavium-jenkins_W3) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/10eef2e489ea38084891ed7b1d2049b559a97eb3/hpc-arm-hwi-jenkins_W0) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/fc36d4d8e608a0fa508d3c5fea72f6a6673ff608/hpc-arm-hwi-jenkins_W2) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/d73fab74fe948e15b7eff5042fa2ecebf19c9eac/hpc-arm-hwi-jenkins_W3) | :question: UNKNOWN
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/4926e626351ba5f0959ce34de8bf8567427e0442/hpc-test-node-gpu_W0) | :question: UNKNOWN
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/b768e0e913fc7fba1ef812f439f1c829fc94d3ad/hpc-test-node-gpu_W1) | :question: UNKNOWN
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/cb1e9b9957898112ebc7feae52614da1a40fb366/hpc-test-node-gpu_W2) | :question: UNKNOWN
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/aef16e2f8ae63c49a087801d840893f6b88274b1/hpc-test-node-gpu_W3) | :question: UNKNOWN
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/980e95aea57293082921b0e69a42a00e3d5d83b2/hpc-test-node-legacy_W0) | :question: UNKNOWN
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/21ee02f971fb96fa7dd94ade76d93342b9968189/hpc-test-node-legacy_W1) | :question: UNKNOWN
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/f8992f0153ecb6e011685d177362c68cccb9d376/hpc-test-node-legacy_W2) | :question: UNKNOWN
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/5ce18d11a575e75486afac003cca9e9cc37ff998/hpc-test-node-legacy_W3) | :question: UNKNOWN
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/b5df8767514a6259824fa429b1350b97038e9072/hpc-test-node-new_W0) | :question: UNKNOWN
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/c51a024fdb08d9188189777d9a6bff2b13cad593/hpc-test-node-new_W1) | :question: UNKNOWN
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/e4769b4c014a9a95f90f54b7ddad4e3cf87bffb2/hpc-test-node-new_W2) | :question: UNKNOWN
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/b27193265d700cd008f7a7a8c21e9a4533ccdb10/hpc-test-node-new_W3) | :question: UNKNOWN
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/6aca382030726410d3a654c4ade706509ed5a2ee/r-vmb-ppc-jenkins_W1) | :question: UNKNOWN
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/df6c5fb069dfb88c2d9383a65d31fb7a7369ceef/r-vmb-ppc-jenkins_W2) | :question: UNKNOWN
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/509e76296b0c33724012dbb86b34e630/raw/c41bfac6ece05f1136bfaa6091ca9a6b66b4a9ae/r-vmb-ppc-jenkins_W3) | :question: UNKNOWN

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 2 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 14-Nov-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/b86f4458499a741885c532f1fb5afad254fe8001/_main) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/68fdcabe6285aeb785e1b933288eead8ef6af188/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/0cf110e9292f31bd48a56e643cf2536574829672/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/a7621202cbb337c21908bbf33ae78be4a916976d/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/52475e6c5196a065d8cfff5ae4699d6668fc333f/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/c9b8750287ea698f0cf97b484ab7d1edf7cf43b1/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/113c456b65e9f5506a0603ebfaf6f6e42d8d33d5/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/01565f85d63753e6145d06a49d8ba775350763aa/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/77fcfd178417ea9f4e8713a83169464487471c27/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/8221b453db3a4bb3620fd396c27eb1661b375289/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/d06054729cf357d07f9d412ae1640dfedaf4d360/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/23e508742318a5af5b791c2cb5d1cd2d4e3c73e1/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/0a8e5fe60146a0714e4ee86f20f16bbb88f04f16/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/50240c5da60c420510325dc5d4eeea6b75bfc79b/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/c842e40a92fc67d3a1445939b55ee64453073f79/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/3c0a689eda0e204cbf4b1b580125fef133414850/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/76075a6f11727217292c11b788cff35f3658d1bc/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/a6d642598f7e60123a2575caf386a12ab31cad22/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/f0a555f789ce621ef51e6c7a2557ef7c5fecb4f8/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/f5785200b0d4ce475160a3be7af7660421fe0e8f/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/fecdd0366b508b82e838e638bf1279b76481d16b/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/483e8bb089f67de17896cad1c132c34d1db2d0d6/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/efed253d7041c411e6f675bcadb307f694b6fd13/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/5d303d26157ce8d8e0d017e300897b74108757a8/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/482a8cbe1a752a595ee4c31ebf834e34/raw/88adccedb383ec5a4ca5f2b75478349be7688a97/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>UNKNOWN</b> on 6 workers (click for details)</summary>

**Note:** the logs will be deleted after 15-Nov-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/9f2157ac3180a8fc0a874f57407753e3/raw/5cf67907c551731acf5c12ef25a19880596259e8/_main) | :question: ABORTED
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/9f2157ac3180a8fc0a874f57407753e3/raw/e7049438c8ba60fb090968ce428867eb253ee927/hpc-arm-cavium-jenkins_W0) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/9f2157ac3180a8fc0a874f57407753e3/raw/8bf17745598a44f2ceeb0b5c4ff634400a360042/hpc-arm-cavium-jenkins_W1) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/9f2157ac3180a8fc0a874f57407753e3/raw/aeccf2eabc5f3f9b09edc7cd8b104d2b70a5e9d8/hpc-arm-cavium-jenkins_W2) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/9f2157ac3180a8fc0a874f57407753e3/raw/c0e1a59820ac345cce8436e8ac5f93d029bb1ebd/hpc-arm-cavium-jenkins_W3) | :question: UNKNOWN
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/9f2157ac3180a8fc0a874f57407753e3/raw/c5fe032020a7aebffd85ff1d7958b90bc5a0e61e/hpc-test-node-legacy_W3) | :question: UNKNOWN

**Comment by brminich** [comment]
bot:mlx:retest
bot:pipe:retest

**Comment by yosefe** [comment]
kernel panic on hpc-test-node-upstream-02
bot:pipe:retest

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 16-Nov-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/88fa71b8a502a23f14861f2248c211cb0507039d/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/81b2ad9e2b13f234f8838bd0b465a54f6d191fc9/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/5feee1db08b397581f84f8486efa79dd9142dd8d/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/fd82b83729ab79ac92aee3c7319f1c671cdfe281/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/da6adb24b078d1a4080a2be047c32068708a2e44/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/016da07c135404b0c29ae59c165a796ab67664c8/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/1d855ed66c7003ebf911f95f18f515467ff786c2/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/8b630c67f8b9102d5b0bd8e128d8c940a9ac790e/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/c47745521bfc0a4ad8631d03854dc233fee58b7b/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/5d639db080154712003fe3467351c1bd66d449ef/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/53b690d490a7f3c9dcfff97c935f66f8d151fca4/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/7e087074b608292d8472fd1f258e7f986e37bbe2/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/c747200829118ec31db801b3a524e97afa5030b6/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/377127e2daf10bed7ec99c7e81f2b00f97b7685e/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/38969a4185c26f7b55ef7bb845eae5b71e554e5a/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/e8971d51a45f9c7ec66b9ca1e6e164492fcebfb2/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/8a75ceaa8c01aec03714980dec4658ac93316890/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/7f0b55878eb95c3ea1c92487848dce7dda2527c5/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/eda54174a57fe03569b020f7590321be2ab8af83/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/2bdf461bf15167768bc333a9128d042f7791332e/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/aa3330dbf8823267baa1cc06416dfc97d80a2b30/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/efb2749f525505b286ab371b45564910ff207163/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/65b3ad2f935e403bea1815c380f1a7efab0649f1/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/050c43753a4dd6e7565d1a6e4b5c9fc68d37f9cd/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/af37cdef3968b5fe5f3de756118798bf/raw/8f135ff94dc78419891edc1624ca08e6f23b6cdc/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by alinask** [comment]
bot:pipe:retest

**Comment by alinask** [comment]
failure is https://github.com/openucx/ucx/issues/4420
bot:pipe:retest

**Pull request merged**